src/pages/Messages/_Star.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Tooltip from '../../components/Tooltip'
+import { Button } from 'antd'
+import {deleteStar, unstarMessage, starMessage, submitStar} from "../../actions/starMessage"
+
+const Star = ({currentUser, isStarred, toggleStar}) => {
+  const handleStarClick = (event) => {
+    if(event){ event.preventDefault() }
+    toggleStar()
+  }
+  return (
+
+    <Tooltip placement='top' title="Star this message">
+      <Button
+        icon={isStarred ? 'star' : 'star-o'}
+        style={isStarred ? {color: '#ffd700'} : {color: '#000000'}}
+        onClick={handleStarClick}
+        />
+    </Tooltip>
+
+  )
```

Comment:
Few formatting tweaks needed here:

1. Empty line before the `return` statement.

2. No empty lines before the JSX code and the `return` parenthesis.

3. Put the `Star.displayName = 'Star'` line before `mapStateToProps`.

4. One empty line between the methods at the bottom, not two.